### PR TITLE
chore: add simple darkmode

### PR DESCRIPTION
**Pull request by onepisYa**
image

image

### CODE REVIEW

**Comment by onepisYa** [comment]
image

White blinded my dog's eyes. add a simple dark mode. hahaha

**Review by dtolnay — CHANGES_REQUESTED** [review]
Please use Bootstrap's builtin dark mode. `filter: invert(80%)` is not appealing as a dark theme.

https://getbootstrap.com/docs/5.3/customize/color-modes/

**Comment by onepisYa** [comment]
> 

rust-quiz now using bootstrap's v4.1 version haven't dark mode.

this is [5.3 customize/color-modes](https://getbootstrap.com/docs/5.3/customize/color-modes/).

but support [theme](https://getbootstrap.com/docs/4.1/getting-started/theming/#available-variables).

**Comment by dtolnay** [comment]
5.3.0-alpha1 is available and supports dark mode.

**Comment by onepisYa** [comment]
>  



> 5.3.0-alpha1 is available and supports dark mode.

Ok. I'm migrating it to v5.3.0 tonight.

**Review by dtolnay — CHANGES_REQUESTED** [review]
It looks like there are tons of unrelated formatting changes, making this PR challenging to review. Could you please revert those, or if you feel strongly that they should be landed, please move non-functional changes to a separate PR from the dark mode one?

**Comment by dtolnay** [comment]
I noticed these buttons in the bottom right are misaligned in Firefox:

Screenshot from 2023-03-12 13-08-09

File: docs/index.html [hunk 1]
```diff
@@ -1,122 +1,128 @@
...
+            <label class="custom-control-label" for="radio-undefined">The program exhibits undefined behavior</label>
           </div>
+          <div class="custom-control custom-radio">
+            <input id="radio-error" name="paymentMethod" type="radio" class="custom-control-input">
+            <label class="custom-control-label" for="radio-error">The program does not compile</label>
+          </div>
+          <div class="custom-control custom-radio">
+            <input id="radio-output" name="paymentMethod" type="radio" class="custom-control-input">
+            <label class="custom-control-label" for="radio-output">
+              <span class="mb-3 mr-2">The program is guaranteed to output:</span>
```
Comment (dtolnay) [inline docs/index.html]: `.mr-2` no longer exists in bootstrap 5.

File: docs/index.html [hunk 2]
```diff
@@ -1,122 +1,128 @@
...
+        <div id="nav">
+          <button id="button-submit" class="btn btn-primary me-2" type="submit">
+            Answer
+          </button><button id="button-hint" class="btn btn-sm btn-outline-secondary me-2" type="button">
+            Hint
+          </button><button id="button-skip" class="btn btn-sm btn-outline-secondary me-2" type="button">
+            Skip
+          </button><button id="button-reveal" class="btn btn-sm btn-outline-secondary me-2" type="button">
+            Reveal<span id="sad"> 😞</span>
+          </button><span id="incorrect" class="me-4 text-danger text-nowrap d-none">
```
Comment (dtolnay) [inline docs/index.html]: Not sure why this is being changed.

File: docs/quiz.css [hunk 3]
```diff
@@ -1,97 +1,131 @@
...
+  }
 }
 
 #accordion:not(.show) {
-    font-size: 0;
-    opacity: 0;
-    transition: opacity .25s, font-size .5s .15s;
+  font-size: 0;
+  opacity: 0;
+  transition: opacity 0.25s, font-size 0.5s 0.15s;
```
Comment (dtolnay) [inline docs/quiz.css:64]: Why is this better, and if it's better, why is it not done consistently for other decimals, like the one you added in `pre code`?

File: docs/index.html (hunk 2 above)
Comment (onepisYa) [reply docs/index.html]: i will repull-request

File: docs/quiz.css (hunk 3 above)
Comment (onepisYa) [reply docs/quiz.css:64]: it's just a fe abbr

**Pull request closed without merging**